exporter/templates/cat1/r3_1/_2.16.840.1.113883.10.20.24.3.2.xml

```diff
@@ -1,11 +1,10 @@
 <entry>
-  <act classCode="ACT" moodCode="EVN" {{ negationIndicator .EntrySection.Entry}}>
+  <act classCode="ACT" moodCode="EVN" {{negationIndicator .EntrySection.Entry}}>
     <!-- Communication from patient to provider -->
-    <templateId root="2.16.840.1.113883.10.20.24.3.2" extension="2014-12-01"/>
+    <templateId root="2.16.840.1.113883.10.20.24.3.2" extension="2016-02-01"/>
     <id root="1.3.6.1.4.1.115" extension="{{.EntrySection.Entry.ObjectIdentifier.ID}}"/>
     {{template "_code.xml" codeDisplayWithPreferredCode .EntrySection.Entry .EntrySection.Entry.Coded .MapDataCriteria "entryCode"}} 
-
-    <text>{{ .EntrySection.Description }}</text>
+    <text>{{.EntrySection.Description }}</text>
```

Comment:
Another leading whitespace clear while missing the trailing one. It's again very minor, but just for consistency's sake while you're nabbing them.